### ISSUE TITLE
Bug: Apps reopen after quitting

### ISSUE THREAD

**Issue by itsnandy** [yuuiko/AquaTouch#71]
**Describe the bug**
A few months now, some applications I close as Mail or Safari would just reopen right after quitting them. I noticed the behavior when BTT was on, but when I quit BTT it stopped /:

**Screenshots**
I have a video that shows exactly what I said, please let me know if you need it

**Details:**
 - MacOS Version: 10.15.7
 - BTT Version: 3.404
 - AQT Version: 3.6.2

**Comment by yuuiko** [comment]
Safari should be better in 3.6.3

**Comment by asingh-eightfold** [comment]
@itsnandy Did you find a fix or something?

**Comment by itsnandy** [comment]
@asingh-eightfold I don't think I've had as many issues with Safari, but Mail and Spotify do sometimes reopen still.

**Comment by asingh-eightfold** [comment]
Yeah, same. Will look for a fix myself then ig... Thanks for the fast reply tho
